quantmsrescore/ms2_model_manager.py

```
@@ -68,6 +104,31 @@ def ms2_fine_tuning(self, psms_df: pd.DataFrame,
         self.epoch_to_train_ms2 = epoch_to_train_ms2
         self.train_ms2_model(psms_df, match_intensity_df)
 
+    def load_installed_models(self, download_model_path: str = "pretrained_models_v3.zip", model_type: str = "generic"):
```

Ruff (0.14.10)

Unused method argument: `model_type`

(ARG002)